src/top-issues/top-issues.service.ts

```diff
@@ -0,0 +1,113 @@
+import { BadRequestException, Injectable, InternalServerErrorException, Logger, NotFoundException } from '@nestjs/common';
+import { CreateTopIssueDto, TopIssueOutputDto, UpdateTopIssueDto } from './schemas/topIssues.schema';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { TopIssue } from '@prisma/client';
+import { PrismaClientValidationError } from '@prisma/client/runtime/library';
+
+@Injectable()
+export class TopIssuesService {
+  private readonly logger = new Logger(TopIssuesService.name);
+  constructor(private prismaService: PrismaService) {}
+
+  async create(body: CreateTopIssueDto): Promise<TopIssueOutputDto> {
+    const { name, icon } = body;
+
+    try {
+      const { id } = await this.prismaService.topIssue.create({
+        data: {
+          name, 
+          icon
+        }
+      })
+
+      return {
+        id,
+        name,
+        icon
+      }
```

Comment:
Prisma `CRUD` operations return the resource that was acted upon:

https://www.prisma.io/docs/orm/prisma-client/queries/crud#create

Might as well use it:
```typescript
return await this.prismaService.topIssue.create({
        data: {
          name, 
          icon
        }
      })
```